internal/credentials/spiffe_test.go

```diff
@@ -231,3 +233,110 @@ func (s) TestSPIFFEIDFromCert(t *testing.T) {
 		})
 	}
 }
+
+var SPIFFETrustBundle map[string]*spiffebundle.Bundle
+
+func TestKnownSPIFFEBundle(t *testing.T) {
+	bundles, err := LoadSPIFFEBundleMap(testdata.Path("spiffe/spiffebundle.json"))
+	if err != nil {
+		t.Fatalf("Error during parsing: %v", err)
+	}
+	if len(bundles) != 2 {
+		t.Fatal("did not parse correct bundle length")
```

Comment:
Strings that are part of log messages or test errors don't have to comply to the error string capitalization style guide recommendation. Please see here: go/go-style/decisions#error-strings. This comment applies to other test error strings in this file as well.